specification/windowsiot/resource-manager/Microsoft.WindowsIoT/preview/2018-02-16-preview/WindowsIotServices.json

```diff
@@ -0,0 +1,680 @@
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DeviceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/DeviceService"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false,
+        "x-ms-examples": {
+          "Services_GetProperties": {
+            "$ref": "./examples/Service_GetProperties.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "DeviceServices"
+        ],
+        "summary": "Create or update the metadata of a Windows IoT Device Service.",
+        "description": "Create or update the metadata of a Windows IoT Device Service. The usual pattern to modify a property is to retrieve the Windows IoT Device Service metadata and security metadata, and then combine them with the modified values in a new body to update the Windows IoT Device Service.",
+        "operationId": "Services_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DeviceNameParameter"
+          },
+          {
+            "name": "deviceService",
+            "in": "body",
+            "description": "The Windows IoT Device Service metadata and security metadata.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DeviceService"
+            }
+          },
+          {
+            "name": "If-Match",
+            "in": "header",
+            "required": false,
+            "type": "string",
+            "description": "ETag of the Windows IoT Device Service. Do not specify for creating a new Windows IoT Device Service. Required to update an existing Windows IoT Device Service."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/DeviceService"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false,
```

Comment:
Please check if this is a long running operation

Reply:
it is not a long running operation.